help/destinations/catalog/advertising/liveramp-onboarding.md

```diff
@@ -64,6 +64,9 @@ To authenticate to the destination, fill in the required fields and select **[!U
 
 ![Sample screenshot showing how to authenticate to the destination using SFTP with password](../../assets/catalog/advertising/liveramp-onboarding/liveramp-sftp-password.png)
 
+* **[!UICONTROL Port]**: The port used for your [!DNL LiveRamp - Onboarding] storage location.  This is contingent on your geographic region.
```

Comment:
"contingent" is one of those terms that should not be present in clear technical documentation. Please rephrase this, maybe like this:

The port used for your [!DNL LiveRamp - Onboarding] storage location.  Use the port that corresponds to your geographical location, as described below:

- `US`: `22`
- `APAC`: `2222`

What about the other regions? What about Australia and EU? What port do those use? Please add those too.

Reply:
Edited to your language.  For the different regions, this will only include North America and Australia, as EU only supports the SSH option.  I've changed the names here to reflect NA and AU to avoid any confusion.